### PR TITLE
STP-3211: UAS formatting and linting

### PR DESCRIPTION
**Pull request by nrockershousen**
## Summary and Scope

General linting of UAS docs. No significant content changes.

- Fixed markdown formatting, spelling, and grammar.
- Removed placeholder files referencing moved content. 

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [STP-3211](https://jira-pro.its.hpecorp.net:8443/browse/STP-3211)
* Change will also be needed in `release/1.2` and `release/1.0`

## Testing

N/A

### Tested on:

N/A

### Test description:

N/A

## Risks and Mitigations

None.

### CODE REVIEW

File: operations/UAS_user_and_admin_topics/Create_a_UAI_Using_a_Direct_Administrative_Command.md [hunk 1]
```diff
@@ -1,3 +0,0 @@
-[Top: User Access Service (UAS)](User_Access_Service_UAS.md)
```
[the remark this answers is not in the document]
Comment (nrockershousen) [reply operations/UAS_user_and_admin_topics/Create_a_UAI_Using_a_Direct_Administrative_Command.md:1]: I did run a search through the other product stream docs to take note of anything that might need to be updated in other repos. Not much is impacted by this change, but I will open a ticket to update the getting started guide and slingshot docs that have references.

**Review by erl-hpe — APPROVED** [review]

File: operations/UAS_user_and_admin_topics/Configure_a_Broker_UAI_Class.md [hunk 2]
```diff
@@ -18,19 +14,25 @@ Configuring a Broker UAI class consists of the following actions:
...
-This example, uses Kubernetes secrets and assumes that the Broker UAIs run in the `uas` Kubernetes namespace. If a different namespace is used, the creation of the ConfigMaps is different but the contents are the same. Using a namespace other than `uas` for Broker UAIs has implications beyond secrets and configmaps, is not recommended and is beyond the scope of this document.
+This example uses Kubernetes secrets and assumes that the Broker UAIs run in the `uas` Kubernetes namespace. If a different namespace is used, the creation of the ConfigMaps is different but the contents are the same.
+Using a namespace other than `uas` for Broker UAIs has implications beyond secrets and ConfigMaps; it is not recommended and is beyond the scope of this document.
 
 1. Configure LDAP and determine which files need to be changed in the Broker UAI and what their contents should be.
 
     In this example, the file is `/etc/sssd/sssd.conf` and its contents are representative but sanitized. Substitute your own site specific contents:
 
-    ```
+    ```bash
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Configure_a_Broker_UAI_Class.md:35]:
```suggestion
    ```text
```

File: operations/UAS_user_and_admin_topics/Configure_a_Broker_UAI_Class.md [hunk 3]
```diff
@@ -100,7 +102,7 @@ This example, uses Kubernetes secrets and assumes that the Broker UAIs run in th
 
      Example output:
 
-     ```
+     ```bash
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Configure_a_Broker_UAI_Class.md:105]:
```suggestion
     ```text
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Configure_a_Broker_UAI_Class.md:105]:
```suggestion
     ```text
```

File: operations/UAS_user_and_admin_topics/Configure_a_Broker_UAI_Class.md [hunk 4]
```diff
@@ -208,7 +210,7 @@ This example, uses Kubernetes secrets and assumes that the Broker UAIs run in th
 
     Example output:
 
-    ```
+    ```bash
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Configure_a_Broker_UAI_Class.md:213]:
```suggestion
    ```text
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Configure_a_Broker_UAI_Class.md:213]:
```suggestion
    ```text
```

File: operations/UAS_user_and_admin_topics/Create_UAIs_From_Specific_UAI_Images_in_Legacy_Mode.md [hunk 5]
```diff
@@ -1,20 +1,16 @@
...
-
-[Next Topic: UAS and UAI Legacy Mode Health Checks](UAS_and_UAI_Health_Checks.md)
-
-## Create UAIs From Specific UAI Images in Legacy Mode
+# Create UAIs From Specific UAI Images in Legacy Mode
 
 A user can create a UAI from a specific UAI image (assuming no default UAI class exists) using a command of the form:
 
-```
+```bash
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_UAIs_From_Specific_UAI_Images_in_Legacy_Mode.md:5]:
```suggestion
```text
```

File: operations/UAS_user_and_admin_topics/Create_a_UAI.md [hunk 6]
```diff
@@ -32,35 +28,41 @@ For Legacy Mode user procedures:
...
-### Procedure
+## Procedure
 
 1. Create a UAI administratively.
 
     Use a command of the following form:
 
-    ```
+    ```bash
     cray uas admin uais create OPTIONS
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI.md:38]:
```suggestion
    ncn# cray uas admin uais create OPTIONS
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI.md:38]:
```suggestion
    linux# cray uas admin uais create OPTIONS
```

File: operations/UAS_user_and_admin_topics/Create_a_UAI.md [hunk 7]
```diff
@@ -32,35 +28,41 @@ For Legacy Mode user procedures:
...
+    UAI names may contain up to 63 lower case alphanumeric or `-` characters, and must start and end with an alphanumeric character.
 
-2. Create a UAI in the Legacy Mode of UAI Management
+2. Create a UAI in the Legacy Mode of UAI Management.
 
     Use a command of the following form:
 
-    ```
+    ```bash
     cray uas create OPTIONS
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI.md:57]:
```suggestion
    ncn# cray uas create OPTIONS
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI.md:57]:
```suggestion
    ncn# cray uas create OPTIONS
```

File: operations/UAS_user_and_admin_topics/Create_a_UAI_Class.md [hunk 8]
```diff
@@ -1,34 +1,32 @@
...
 * The administrator must be logged in as an administrator to the HPE Cray EX System CLI (`cray auth login` command)
 
-### Procedure
+## Procedure
 
 Add a UAI class by using the command in the following example.
 
-```
+```bash
 ncn-m001-pit# cray uas admin config classes create --image-id <image-id> [options]
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI_Class.md:17]:
```suggestion
ncn# cray uas admin config classes create --image-id <image-id> [options]
```
```suggestion
ncn-m001-pit# cray uas admin config classes create --image-id <image-id> [options]
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI_Class.md:17]:
```suggestion
ncn# cray uas admin config classes create --image-id <image-id> [options]
```
```suggestion
ncn-m001-pit# cray uas admin config classes create --image-id <image-id> [options]
```

File: operations/UAS_user_and_admin_topics/Create_a_UAI_Class.md [hunk 9]
```diff
@@ -1,34 +1,32 @@
...
 
-The only required option is `--image-id IMAGE_ID` which sets the container image that will be used to create a UAI from this UAI Class.
+The only required option is `--image-id IMAGE_ID` which sets the container image that will be used to create a UAI from this UAI class.
 
-Other options and arguments can be discovered using the:
+Other options and arguments can be discovered using the following command:
 
-```
+```bash
 ncn-m001-pit# cray uas admin config classes create --help
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI_Class.md:25]:
```suggestion
ncn# cray uas admin config classes create --help
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI_Class.md:25]:
```suggestion
ncn# cray uas admin config classes create --help
```

File: operations/UAS_user_and_admin_topics/Create_a_UAI_Resource_Specification.md [hunk 10]
```diff
@@ -1,38 +1,39 @@
...
-### Procedure
+## Procedure
 
 Add a resource specification.
 
 Use a command of the following form:
 
-```
+```bash
 ncn-m001-pit # cray uas admin config resources create [--limit <k8s-resource-limit>] [--request <k8s-resource-request>] [--comment '<string>']
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI_Resource_Specification.md:20]:
```suggestion
ncn# cray uas admin config resources create [--limit <k8s-resource-limit>] [--request <k8s-resource-request>] [--comment '<string>']
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI_Resource_Specification.md:20]:
```suggestion
ncn# cray uas admin config resources create [--limit <k8s-resource-limit>] [--request <k8s-resource-request>] [--comment '<string>']
```

File: operations/UAS_user_and_admin_topics/Create_a_UAI_Resource_Specification.md [hunk 11]
```diff
@@ -1,38 +1,39 @@
...
 ```
 
 For example:
 
-```
+```bash
 ncn-m001-pit# cray uas admin config resources create --request '{"cpu": "300m", "memory": "1Gi"}' --limit '{"cpu": "300m", "memory": "1Gi"}' --comment "Resource Specification to use with Brokered End-User UAIs"
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI_Resource_Specification.md:26]:
```suggestion
ncn# cray uas admin config resources create --request '{"cpu": "300m", "memory": "1Gi"}' \
            --limit '{"cpu": "300m", "memory": "1Gi"}' \
            --comment "Resource Specification to use with Brokered End-User UAIs"
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI_Resource_Specification.md:26]:
```suggestion
ncn# cray uas admin config resources create --request '{"cpu": "300m", "memory": "1Gi"}' \
            --limit '{"cpu": "300m", "memory": "1Gi"}' \
            --comment "Resource Specification to use with Brokered End-User UAIs"
```

File: operations/UAS_user_and_admin_topics/Create_a_UAI_with_Additional_Ports.md [hunk 12]
```diff
@@ -54,5 +56,7 @@ Only ports 80, 443, and 8888 can be exposed. Attempting to open any other ports
 2. Log in to the UAI with the connection string.
 
     ```bash
-    $ ssh USERNAME@UAI_IP_ADDRESS -i ~/.ssh/id_rsa
+    ssh USERNAME@UAI_IP_ADDRESS -i ~/.ssh/id_rsa
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI_with_Additional_Ports.md:59]:
```suggestion
    linux# ssh USERNAME@UAI_IP_ADDRESS -i ~/.ssh/id_rsa
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_a_UAI_with_Additional_Ports.md:59]:
```suggestion
    linux# ssh USERNAME@UAI_IP_ADDRESS -i ~/.ssh/id_rsa
```

File: operations/UAS_user_and_admin_topics/Create_and_Use_Default_UAIs_in_Legacy_Mode.md [hunk 13]
```diff
@@ -76,7 +72,7 @@ username = "vers"
 
 Using `cray uas list`, the user watches the UAI until it reaches a `Running: Ready` state. The UAI is now ready to accept SSH logins from the user, and the user then logs into the UAI to run a simple Slurm job, and logs out.
 
-```
+```bash
 vers>  ssh vers@34.136.140.107
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_and_Use_Default_UAIs_in_Legacy_Mode.md:76]:
```suggestion
vers> ssh vers@34.136.140.107
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Create_and_Use_Default_UAIs_in_Legacy_Mode.md:76]:
```suggestion
vers> ssh vers@34.136.140.107
```

File: operations/UAS_user_and_admin_topics/Customize_the_Broker_UAI_Image.md [hunk 14]
```diff
@@ -237,13 +238,15 @@ The following is an example of replacing the entrypoint script with a new entryp
...
+    ```bash
+    ncn-m001-pit# cray uas admin config classes update \
+    --volume-list '11a4a22a-9644-4529-9434-d296eef2dc48,1ec36af0-d5b6-4ad9-b3e8-755729765d76,2246bbb1-4006-4b11-ba57-6588a7b7c02f,a3b149fd-c477-41f0-8f8d-bfcee87fdd0a' \
+    d764c880-41b8-41e8-bacc-f94f7c5b053d --format yaml
     ```
 
     Example output:
 
-    ```
+    ```bash
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Customize_the_Broker_UAI_Image.md:249]:
```suggestion
    ```text
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Customize_the_Broker_UAI_Image.md:249]:
```suggestion
    ```text
```

File: operations/UAS_user_and_admin_topics/Customize_the_Broker_UAI_Image.md [hunk 15]
```diff
@@ -300,13 +303,14 @@ The following is an example of replacing the entrypoint script with a new entryp
...
+   and the existing Broker UAI (if any) and create a new Broker UAI.
 
     **NOTE:** Clearing out existing UAIs will terminate any user activity on those UAIs, make sure that users are warned of the disruption.
 
-    1.  Clear out the UAIs.
+    1. Clear out the UAIs.
 
-        ```
+        ```bash
         ncn-m001-pit# cray uas admin uais delete --class-id bdb4988b-c061-48fa-a005-34f8571b88b4
```
Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Customize_the_Broker_UAI_Image.md:315]:
```suggestion
```

Comment (mharding-hpe) [inline operations/UAS_user_and_admin_topics/Customize_the_Broker_UAI_Image.md:315]:
```suggestion
```

**Review by mharding-hpe — APPROVED** [review]

**Pull request merged**